data/01_data_prep_dashboard.R

```diff
@@ -0,0 +1,54 @@
+# Gradients Metropool Amsterdam
+#
+# Edit data for the dashboard
+
+
+# create workspace
+rm(list=ls())
+options(scipen=999)
+
+
+#### PACKAGES ####
+library(tidyverse)
+library(readxl)
+
+# add NL name
+outcome_tab <- read_excel("~/Documents/GitHub/kenniscentrumongelijkheid/data/nl/outcome_table.xlsx") 
```

Comment:
Remove part of the hard-coded file paths. This also applies to the other file paths.